sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs

```diff
@@ -36,4 +36,4 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.1.0.0")]
```

Comment:
```suggestion
[assembly: AssemblyFileVersion("0.10.0.0")]
```

Reply:
@isra-fel 
update to
[assembly: AssemblyVersion("0.10.0.0")]
[assembly: AssemblyFileVersion("0.10.0.0")]